src/components/Navbar.js

```diff
@@ -1,20 +1,37 @@
 import React from "react";
 import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import history from "../assets/icons/history.svg";
```

Comment:
Might be nicer to change these variables to, e.g. `historyIcon`, so it's can't be confused with a component.

Reply:
done